package.json

```diff
@@ -1,26 +1,36 @@
 {
-  "name": "leader-board-api",
+  "name": "to-do-list",
```

Comment:
Good job by correctly configuring web pack 👍🏼  but why did you change the project name here?!!